packages/js-recommendations/README.md

```diff
@@ -119,11 +147,16 @@ The translations for the component.
 <summary><code>(props: ChildrenProps) => JSX.Element</code></summary>
 
 ```ts
-type ChildrenProps<TObject> = {
+type ComponentProps<TObject> = RendererProps & {
```

Comment:
We don't reference `RendererProps` so perhaps we can inline `createElement` and `Fragment`? (without theirs docs)

Reply:
Make sense, fixed in https://github.com/algolia/ui-components/pull/26/commits/b23520aab7b52799a2dced3ba5ab589871a4ff35